### PR TITLE
[CodeGen] NFC: Rename LinalgTileAndFusePass

### PR DESCRIPTION
**Pull request by antiagainst**
This pass is only for tiling and vectorization within a single
workgroup right now. Rename to make it clear.

### CODE REVIEW

File: iree/compiler/Conversion/LinalgToSPIRV/BUILD [hunk 1]
```diff
@@ -41,10 +41,10 @@ cc_library(
         "FoldGPUProcessorIDUses.cpp",
         "KernelDispatchUtils.cpp",
         "LinalgTileAndDistributePass.cpp",
-        "LinalgTileAndFusePass.cpp",
         "MatMulVectorizationTest.cpp",
         "Passes.cpp",
         "SplitDispatchFunctionPass.cpp",
+        "TileAndVectorizeInOneWorkgroupPass.cpp",
```
Comment (MaheshRavishankar) [inline iree/compiler/Conversion/LinalgToSPIRV/BUILD:47]: Just make this `TileAndVectorizePass`. Why in one workgroup?

**Review by MaheshRavishankar — CHANGES_REQUESTED** [review]

File: iree/compiler/Conversion/LinalgToSPIRV/BUILD (hunk 1 above)
Comment (antiagainst) [reply iree/compiler/Conversion/LinalgToSPIRV/BUILD:47]: It's emphasizing that his is not performing the first-level tiling. It is also a nice contrast to `ConcretizeTileAmongWorkgroupsPass`. :)

Comment (MaheshRavishankar) [reply iree/compiler/Conversion/LinalgToSPIRV/BUILD:47]: I dont like the new name :D , but the one thing worse than API wars is naming wars, so I will not harp on this anymore.

**Review by MaheshRavishankar — APPROVED** [review]

File: iree/compiler/Conversion/LinalgToSPIRV/BUILD (hunk 1 above)
Comment (antiagainst) [reply iree/compiler/Conversion/LinalgToSPIRV/BUILD:47]: Hehe. Yeah I agree it's subjective. But it should be an improvement over the current name I believe. :D

**Pull request merged**